generators/server/templates/src/main/resources/_logback-spring.xml

```diff
@@ -120,11 +112,4 @@
         <resetJUL>true</resetJUL>
     </contextListener>
 
-    <root level="#logback.loglevel#">
```

Comment:
@PierreBesson @jdubois may be the blcok should not be deleted, is CONSOLE the default appender ref for root? Coz when I did this on one of my app where I use a custom console appender like below it doesnt log anything
```
<appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
            <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
                <layout class="com.**.StackdriverLogbackLayout">
                    <pattern>${FILE_LOG_PATTERN}</pattern>
                    <serviceName>${SERVICE_NAME}</serviceName>
                    <serviceVersion>${SERVICE_VERSION}</serviceVersion>
                </layout>
            </encoder>
        </appender>
```

If I add 
```
    <root>
        <appender-ref ref="CONSOLE"/>
    </root>
```
it works

Reply:
Yes it's supposed to be the default, following the 12 Factors... But I haven't checked in a while. Now that's configured in the application-*.yml files, did you check them?

Reply:
@jdubois @kaidohallik Also I realized that when setting these in an environment variable to overide it doesn't work
I tried the all of below and they had no effect. I'll probably create an issue in spring boot
```
LOGGING_LEVEL_COM_MYCOMP=INFO
_JAVA_OPTIONS=-Dlogging.level.com.mycomp=INFO
SPRING_APPLICATION_JSON={ "logging.level.com.mycomp": "INFO" }
```

Reply:
@deepu105 It doesn't work this way. Those loggers configs are stored in a spring boot `configuration properties class` as maps of which "com.mycomp.*" is a key. And the way you are accessing it, you are requesting the following property : `logging.level.com.mycomp` which doesn't exist.
I checked and you can set it this way:
```
LOGGING_LEVEL_com.netflix.discovery=WARN
_JAVA_OPTIONS=-Dlogging.level."com.netflix.discovery"=WARN
// didn't make it work for the JSON
```
I agree that it's not straightforward. We should write this in our docs.

Reply:
yes +1 for documenting this!

Reply:
@PierreBesson I tried both and doesnt work, first one doesnt normally work due to the . in the name, I can do a workaround but dont want to rely on that as I need to do this in my production environments. For the second one its still picking up without the quotes
```
export _JAVA_OPTIONS=-Dlogging.level."com.mycomp"=WARN && ../gradlew
Picked up _JAVA_OPTIONS: -Dlogging.level.com.mycomp=WARN
```

Reply:
Oh wait I didnt read it properly let me try
`com.mycomp.something`

Reply:
Ok it indeed works after third level :) I wish these where documented properly by Spring